src/cli/onefuzz/api.py

```diff
@@ -849,6 +849,7 @@ def create(
         colocate: bool = False,
         report_list: Optional[List[str]] = None,
         minimized_stack_depth: Optional[int] = None,
+        coverage_filter: Optional[str] = None,
```

Comment:
This should be an Optional[File].  See target_exe earlier.  This allows us to verify the file exists as we expect.

Reply:
I don't see where coverage_filter gets uploaded in the case we don't have a setup directory specified on the command line.

Reply:
Ah, right, I'll reuse that pattern. I was looking at `_create_tasks()`, where we've already leveraged the `File` checks, and cast to a `str`.

Re the upload, we don't do that right now, I was originally thinking we _shouldn't_ upload it, if it isn't in the `setup` container (though we should check that it is there). But I forgot that we already do this as-needed for target-associated `.pdb`, so we should upload the filter, too.

Reply:
I'm going to first rename a helper function in a separate PR, because it will be a bit noisy.

Reply:
Rename split out into #926.